File: crates/mun_runtime/benches/resources/fibonacci.lua
function fibonacci(n)
    a, b, i = 0,0,0
    repeat
        if i > n then
            return a
        end
        sum = a+b
        a,b = b,sum
        i = i + 1
    until false
end

function main(n)
    return fibonacci(n)
end

File: crates/mun_runtime/benches/resources/fibonacci.rs
pub fn fibonacci(n: i64) -> i64 {
    let mut a = 0;
    let mut b = 1;
    let mut i = 1;
    loop {
        if i > n {
            return a;
        }
        let sum = a + b;
        a = b;
        b = sum;
        i += 1;
    }
}

pub fn fibonacci_main(n:i64) -> i64 {
    fibonacci(n)
}